composer.json

```diff
@@ -21,36 +21,39 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "phpcr/phpcr-utils": "^1.3",
-        "symfony/doctrine-bridge": "^5.4 || ^6.0",
-        "symfony/framework-bundle": "^5.4 || ^6.0",
-        "symfony/cache": "^5.4 || ^6.0"
+        "php": "^8.0",
+        "phpcr/phpcr-utils": "^1.3 || ^2.0",
+        "symfony/cache": "^5.4 || ^6.0 || ^7.0",
+        "symfony/doctrine-bridge": "^5.4 || ^6.0 || ^7.0",
+        "symfony/framework-bundle": "^5.4 || ^6.0 || ^7.0"
     },
     "conflict": {
         "doctrine/annotations": "< 1.7.0",
         "doctrine/doctrine-bundle": "< 2.0.3",
         "jackalope/jackalope": "< 1.3.1",
-        "phpcr/phpcr-shell": "< 1.0.0-beta1"
+        "phpcr/phpcr-shell": "< 1.0.0-beta1",
+        "symfony/dependency-injection": "< 3",
+        "symfony/console": "< 4"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^2.0.3",
-        "doctrine/phpcr-odm": "^2.0",
-        "jackalope/jackalope-doctrine-dbal": "^1.3 || ^2.0",
-        "matthiasnoback/symfony-dependency-injection-test": "^4.1",
-        "symfony/asset": "^5.4 || ^6.0",
-        "symfony/browser-kit": "^5.4 || ^6.0",
-        "symfony/css-selector": "^5.4 || ^6.0",
-        "symfony/error-handler": "^4.4 || ^5.0  || ^6.0",
-        "symfony/form": "^5.4 || ^6.0",
-        "symfony/monolog-bridge": "^5.4 || ^6.0",
+        "doctrine/phpcr-odm": "dev-allow-persistence-3 as 2.0.0",
```

Comment:
don't forget to change to ^2.0 when https://github.com/doctrine/phpcr-odm/pull/860 is merged